### PR TITLE
✨ Download all pages for social graph csv export

### PR DESCRIPTION
**Pull request by williamchong**
TODO: fix indexer api that returns duplicated owner address in different pages

### CODE REVIEW

File: src/pages/SocialGraph.vue [hunk 1]
```diff
@@ -270,9 +290,31 @@ export default {
...
+        if (!isPageEnd) this.allPageData = this.allPageData.concat(data);
+        i += 1;
+      }
+      this.allPageData.sort(
+        (a, b) => b.totalValue - a.totalValue,
+      );
+
+      // TODO: Fix indexer API to not return duplicated addresses
+      this.allPageData = this.allPageData
+        .filter((c, index) => this.allPageData.indexOf(c) === index);
```
Comment (nwingt) [inline src/pages/SocialGraph.vue]:
```suggestion
      this.allPageData = [...new Map(this.allPageData.map(c => [c.account, c])).values()];
```

Comment (williamchong) [reply src/pages/SocialGraph.vue]: sorcery... updated

**Review by nwingt — APPROVED** [review]

File: src/pages/SocialGraph.vue (hunk 1 above)
Comment (WeiJun0827) [reply src/pages/SocialGraph.vue]: If two or more items in the `allPageData` has the same `account`, will the item with the lowest `totalValue` remain in the `allPageData`, right?

Comment (williamchong) [reply src/pages/SocialGraph.vue]: the strange thing is indexer returns the exact same content more than once for some address in different pages, so the dropped version should be same as kept one

**Pull request merged**